docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -147,6 +151,47 @@ The API ML TLS requires servers to provide HTTPS ports. Each of the API ML servi
     - Authentication is service-dependent
     - Recommended to use the Authentication and Authorization Service for authentication
 
+#### Authentication endpoints
+
+The API Gateway contains two REST API authentication endpoints: `auth/login` and `auth/query`.
+
+The `/login` endpoint allows to authenticate mainframe user credentials and returns an authentication token. The login request requires the user credentials in one of the following formats:
+  * Basic access authentication
+  * JSON with user credentials 
+
+When successfully authenticated, the response to the request is an empty body and a token in a secure `HttpOnly` cookie named `apimlAuthenticationToken`, otherwise it returns a 401 status code.
+
+The `/query` endpoint allows to validate the token and retrieves the information associated with the token.
+The query request requires the token in one of the following formats: 
+  * Cookie named `apimlAuthenticationToken`
+  * Bearer authentication  
+  
+When successfully authenticated, the response to the request is a JSON object, which contains information associated with the token, otherwise it returns a 401 status code. 
+
+#### z/OSMF Authentication Provider 
```

Comment:
I would create a chapter about Providers in general. Say that we currently have 2 and which one to use. Something really short because details are in the following sub-chapters.